UNRELEASED.md

```diff
@@ -10,3 +10,6 @@
          * Some bug fix, see #124
 
      DO NOT LEAVE A BLANK LINE BELOW THIS PREAMBLE -->
+### Features
+
+ * `RECURSIVE` operators and functions are no longer supported, see #1565
```

Comment:
1. Isn't this is a breaking change and not a feature?
2. Shouldn't the release notes update be saved for when we actually make the change to the software, not just to the docs?